django_cloud_deploy/skeleton/requirements_parser.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""A module help parse requirements.txt."""
+import os
+import re
+from typing import Set
+
+
+def parse_line(line: str) -> str:
+    """Parse a single line in requirements.txt.
+
+    Note that this function only returns what are the requirements. It will not
```

Comment:
I think that the word `requirements` is confusing here. Maybe

Note that this function only returns the package that is required, without version restrictions e.g. "google-cloud-bigquery>1.2.3" => "google-cloud-bigquery".